.github/workflows/markdown-link-check-pr.yaml

```diff
@@ -0,0 +1,38 @@
+name: PR check Markdown links
```

Comment:
Separating this into its own workflow is intentional. The reason for it is that we want to take advantage of parallelism and restrict the set of events that are considered.
The same applies to the workflow for the periodic execution